core/client.go

```diff
@@ -64,11 +64,15 @@ func CreateClients(src, dst *ProvableChain, srcHeight, dstHeight exported.Height
 	// Send msgs to both chains
 	if clients.Ready() {
 		// TODO: Add retry here for out of gas or other errors
-		if clients.Send(src, dst); clients.Success() {
+		srcMsgIDs, dstMsgIDs := clients.Send(src, dst)
+		if clients.Success() {
 			logger.Info(
 				"★ Clients created",
 			)
 		}
+		if err := SyncChainConfigsFromEvents(pathName, srcMsgIDs, dstMsgIDs, src, dst, ConfigIDClient); err != nil {
```

Comment:
Do you intentionally call `SyncChainConfigsFromEvents`, even if `clients.Success() == false`?
Even if the src side fails, if the dst side succeeds, we still need to save the dst side client ID to the config file.
Do you mean this?

Reply:
Check success